tests/plugins/imagebase/features/manual/captionemptyplaceholder.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.9.1, bug, 1776
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, basicstyles, imagebase
+@bender-include: %BASE_PATH%/plugins/easyimage/_helpers/tools.js
+
+1. Focus first image.
```

Comment:
I'm not sure if this procedure is enough to reproduce the bug. It seems that typing and deleting – as stated in #1776 - is necessary.

Reply:
The HTML of this manual test already have empty `figcaption` element to reproduce removing caption content. I reproduced manual test steps without my changes and the test was broken so I think it works fine. I created it to shorten test steps but if you think it's too terse I will refactor the test.

Reply:
Strangely, I could reproduce it in your test, but no success in `/tests/plugins/imagebase/features/manual/caption` 🤔